test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
+  for (const [shortname, json] of Object.entries(all)) {
+    describe(`The ${shortname} entry in @webidl/css`, () => {
+      for (const { type, prop, value } of cssValues) {
+        if (!json[prop]) {
+          continue;
+        }
+        for (const [name, desc] of Object.entries(json[prop])) {
+          if (!desc[value]) {
```

Comment:
I've checked that this can actually happen, but I'm not sure when or if it's intentional. See above question.

Reply:
There are several definitions for which the value is defined in the prose. In such cases, all we have is a `prose` key, for example, see result for [`<timeline-name`](https://github.com/w3c/webref/blob/master/ed/css/css-animations-2.json#L41).

Also, `newValues` is used when the definition extends a base definition.

There should always be a `value`, `newValues` or `prose` key. I'll add a test to assert that this is indeed the case.

Reply:
Great, thank you!

Reply:
So, that's not always the case for now. The exceptions to the rule are mostly `-webkit-` prefixed properties and legacy properties that are defined in prose but for which we currently don't extract more than the property name:
- the 62 [`-webkit-` aliases properties](https://compat.spec.whatwg.org/#css-simple-aliases) extracted from the compat spec
- the 3 [legacy group properties](https://drafts.csswg.org/css-align/#gap-legacy) in css-align
- [`-webkit-line-clamp`](https://drafts.csswg.org/css-overflow/#propdef--webkit-line-clamp) in css-overflow
- [`-webkit-user-select`](https://drafts.csswg.org/css-ui/#propdef--webkit-user-select) in css-ui
- [`stop-color`](https://svgwg.org/svg2-draft/pservers.html#StopColorProperty) and [`stop-opacity`](https://svgwg.org/svg2-draft/pservers.html#StopOpacityProperty) in SVG

I'd be tempted to treat that separately after this PR lands.

Reply:
I created a couple of issues to track that: #126 and https://github.com/w3c/reffy/issues/538